backend/accounts/migrations/0002_auto_20210213_1649.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.1.5 on 2021-02-13 15:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='customuser',
+            options={},
+        ),
+        migrations.AlterField(
+            model_name='customuser',
+            name='email',
+            field=models.EmailField(max_length=254, unique=True, verbose_name='email address'),
+        ),
+        migrations.AlterModelTable(
+            name='customuser',
+            table='auth_user',
+        ),
+    ]
```

Comment:
Jak uz jsem psal v komentu u modelu -- pokud to pregenerujes, zadna druha migrace nebude.

(uzivejme si komfortu pri nasazovani novych aplikaci; tj. jeste nikde nejsou, nepotrebujem drzet zpetne kompatibilitu a navazovat na jiz existujici migrace; vykopnuti nove appky znamena, ze bychom meli mit jen jednu jedinou migraci definujici schema a potencionalne par dalsich datovych pokud to je treba)

Reply:
> Jak uz jsem psal v komentu u modelu -- pokud to pregenerujes, zadna druha migrace nebude.
> 
> (uzivejme si komfortu pri nasazovani novych aplikaci; tj. jeste nikde nejsou, nepotrebujem drzet zpetne kompatibilitu a navazovat na jiz existujici migrace; vykopnuti nove appky znamena, ze bychom meli mit jen jednu jedinou migraci definujici schema a potencionalne par dalsich datovych pokud to je treba)

Super komentář :) Docela jsem s tím zápasil. Vygeneruju migrace znovu.